File: SDKs/googleapis.com/container/v1/python/src/sdk/models/shared/nodetaints.py
from dataclasses import dataclass, field
from typing import List,Optional
from dataclasses_json import dataclass_json
from sdk import utils
from . import *


@dataclass_json
@dataclass
class NodeTaints:
    r"""NodeTaints
    Collection of Kubernetes [node taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration).
    """
    
    taints: Optional[List[NodeTaint]] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('taints') }})
    

File: SDKs/googleapis.com/container/v1beta1/typescript/src/sdk/models/shared/nodetaints.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import { NodeTaint } from "./nodetaint";



// NodeTaints
/** 
 * Collection of Kubernetes [node taints](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration).
**/
export class NodeTaints extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=taints", elemType: NodeTaint })
  taints?: NodeTaint[];
}
